templates/email/default/waste/_bulky_data.html

```diff
@@ -2,11 +2,21 @@
     [% SET sent_confirm_id_ref = report.id %]
 [% END %]
 
+[% IF cobrand.moniker == 'sutton' || cobrand.moniker == 'kingston' %]
```

Comment:
Do we see a downside to including the day and year all the time? It might be nice to have the day, and I guess the year can't hurt, especially if it was a few weeks off maybe? I'd even perhaps be persuaded to have it the default everywhere, not just in the email - that'd be the summary page, the bin days page, and the confirmation page I guess, hmm. What do you think?

Reply:
I suppose the only very minor risk is that it might force a new line on a web page, though not really that much longer and don't see that should really be an issue. I'd certainly appreciate the day being there if I was reading the date.